go.mod

```diff
@@ -1,12 +1,17 @@
 module github.com/terraform-providers/terraform-provider-kubernetes
 
 require (
+	github.com/Azure/go-autorest/autorest v0.9.2 // indirect
+	github.com/Azure/go-autorest/autorest/adal v0.8.1-0.20191028180845-3492b2aff503 // indirect
 	github.com/frankban/quicktest v1.4.2 // indirect
 	github.com/gogo/protobuf v1.3.0 // indirect
 	github.com/google/go-cmp v0.3.1
 	github.com/googleapis/gnostic v0.2.0 // indirect
 	github.com/gophercloud/gophercloud v0.3.1-0.20190807175045-25a84d593c97 // indirect
+	github.com/hashicorp/go-getter v1.4.2-0.20200106182914-9813cbd4eb02 // indirect
 	github.com/hashicorp/go-version v1.2.0
+	github.com/hashicorp/hcl/v2 v2.3.0 // indirect
```

Comment:
I'm curious what introduced these dependency changes. It doesn't look like the new code adds any unusual new imports.

Reply:
🤔 I ran `go mody tidy` but nothing changed, not sure what's going on here.